lib/nailed/github.rb

```diff
@@ -1,14 +1,75 @@
+#
+# Nailed::Github
+#
 module Nailed
+  
+  #
+  # github helpers
+  #
+  def get_org_repos(github_client, org)
+    all_repos = github_client.org_repos(org)
+    all_repos.map(&:name)
+  end
+
+  def fill_db_after_migration(github_client)
```

Comment:
I deliberately moved this to `lib/nailed/github.rb` to give it context. `Nailed.get_org_repos` doesn't tell you which org and which repos. `Nailed::Github.get_org_repos` is much clearer.